app/__init__.py

```diff
@@ -35,6 +35,11 @@ def create_app(config_name):
     login_manager.login_view = 'main.render_login'
     main_blueprint.config = application.config.copy()
 
+    @application.before_request
+    def remove_trailing_slash():
+        if request.path.endswith('/'):
+            return redirect(request.path[:-1], code=301)
```

Comment:
@allait I think these comments are relevant to the buyer app only at the moment. Separate pull request here: https://github.com/alphagov/digitalmarketplace-buyer-frontend/pull/53